services/state/addressbook/dynamic_address.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import "basic_types.proto";
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.token">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * Representation of a Dynamic Node address in the network Merkle tree
+ *
+ * As with all network entities, a DynamicAddress has a unique nodeId, which is usually given along
+ */
+
+message DynamicAddress {
+    /**
+     * The unique entity id of the DynamicAddress.
+     */
+    int64 node_id = 1;
```

Comment:
I don't think these are all properties that need to be stored. Do we have a document listing all the entities we need to fill in data (config.txt, file 101/102, our current internal data structure) and their properties? This would make it easier to see if we thought of all.

Reply:
It would be good to reference the document you mentioned here. Do we have this document @tinker-michaelj @Neeharika-Sompalli ?

Reply:
I created doc https://www.notion.so/swirldslabs/Internal-Data-Structure-of-config-txt-and-file-101-102-ca0e56a06c87474ea543d596b5363605?pvs=4